packages/frontend/src/components/SchedulersView/index.tsx

```diff
@@ -0,0 +1,121 @@
+    const { showToastSuccess } = useToaster();
+    const emptyState = (
+        <ResourceEmptyState
+            title="No scheduled deliveries on this project"
+            description="Go to a chart or dashboard to set up your first scheduled delivery"
+        />
+    );
+    const logsEmptyState = (
+        <ResourceEmptyState
+            title="Scheduled deliveries have not run any jobs as of now"
+            description="Check in later of hit the refresh button to see if any jobs have run"
+        />
+    );
+    return (
+        <SettingsCard style={{ overflow: 'visible' }} p={0} shadow="none">
+            <Tabs
+                classNames={tableTabStyles.classes}
+                keepMounted={false}
+                defaultValue="scheduled-deliveries"
+            >
+                <Group
+                    align="center"
+                    pr="md"
+                    spacing="xs"
+                    sx={{
+                        flexGrow: 1,
+                    }}
+                >
+                    <Tabs.List>
+                        <Tabs.Tab
+                            value="scheduled-deliveries"
+                            icon={
+                                <MantineIcon
+                                    icon={IconSend}
+                                    size="md"
+                                    color="gray.7"
+                                />
+                            }
+                        >
+                            <Title order={6} fw={500} color="gray.7">
+                                All schedulers
+                            </Title>
+                        </Tabs.Tab>
+                        <Tabs.Tab
+                            value="run-history"
+                            icon={
+                                <MantineIcon
+                                    icon={IconClock}
+                                    size="md"
+                                    color="gray.7"
+                                />
+                            }
+                        >
+                            <Title order={6} fw={500} color="gray.7">
+                                Run history
+                            </Title>
+                        </Tabs.Tab>
+                    </Tabs.List>
+                    <Tooltip label="Click to refresh the status of the scheduled deliveries">
+                        <ActionIcon
+                            ml="auto"
+                            onClick={() =>
+                                queryClient
+                                    .invalidateQueries('schedulerLogs')
+                                    .then(() =>
+                                        showToastSuccess({
+                                            title: 'Scheduled deliveries refreshed successfully',
+                                        }),
+                                    )
+                            }
```

Comment:
I would extract this above in the component level as a handler function `handleRefresh`